src/network/netdev/xfrm.c

```diff
@@ -0,0 +1,71 @@
+/* SPDX-License-Identifier: LGPL-2.1+ */
+
+#include <net/if.h>
+
+#include "netdev/xfrm.h"
+
+static int xfrm_config_verify(NetDev *netdev, const char *filename) {
+        Xfrm *x;
+
+        assert(netdev);
+        assert(filename);
+
+        x = XFRM(netdev);
+
+        if (!x->dev) {
+                log_warning("Xfrm NetDev without dev set in %s. Defaulting to 'lo'", filename);
```

Comment:
The default is documented. So, I think it is not necessary to warn about that. Dropping the log or the following is enough
```
log_info("Xfrm NetDev without Device= in %s. Using 'lo'", filename);
```